From 63fda3423c2cf9d5ef104918dba6cef92eaf79a5 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:09:48 +0800
Subject: [PATCH 0161/5242] MLK-11123-24 ARM: dts: disable otg features for
 imx6sl-evk board

commit  2f9be80c2998ab670100f682bb5bc4a4914f3820 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable all otg features for otg1 port of imx6sl-evk board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit 855c996b52317760e114e7884810a0b438b6e3ee)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 92ad01f..26dc3be 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -585,6 +585,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usbotg1>;
 	disable-over-current;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

